app/models/miq_report_result.rb

```diff
@@ -184,7 +184,10 @@ def async_generate_result(result_type, options = {})
     task.update_status("Queued", "Ok", "Task has been queued")
 
     sync = ::Settings.product.report_sync
-
+    # Queue Pattern, Back ground job
+    # Queue_name should have been reporting
```

Comment:
I think you want to change the queue_name to `"reporting"` instead of the comment here.